app/src/main/java/org/mozilla/fenix/ext/BrowserState.kt

```diff
@@ -9,24 +9,36 @@ import mozilla.components.browser.state.selector.selectedNormalTab
 import mozilla.components.browser.state.state.BrowserState
 import mozilla.components.browser.state.state.TabSessionState
 import mozilla.components.feature.tabs.ext.hasMediaPlayed
+import org.mozilla.fenix.FeatureFlags
+import org.mozilla.fenix.home.recenttabs.RecentTab
+import org.mozilla.fenix.tabstray.browser.TabGroup
+import org.mozilla.fenix.tabstray.browser.maxActiveTime
+import org.mozilla.fenix.tabstray.ext.isNormalTabActiveWithSearchTerm
+import kotlin.math.max
 
 /**
  * Get the last opened normal tab and the last tab with in progress media, if available.
  *
- * @return A list of the last opened tab and the last tab with in progress media
+ * @return A array list of the last opened tab and the last tab with in progress media
```

Comment:
```suggestion
 * @return An array list of the last opened tab and the last tab with in progress media
```

Reply:
Thanks

Reply:
I'll update the comment here.  Reverting back to using a list as well.